app/gui/SettingsView.qml

```diff
@@ -708,15 +708,49 @@ Flickable {
                 anchors.fill: parent
                 spacing: 5
 
-                CheckBox {
-                    id: startMaximizedCheck
+                Label {
                     width: parent.width
-                    text: qsTr("Maximize Moonlight window on startup")
+                    id: uiDisplayModeTitle
+                    text: qsTr("Display Mode")
                     font.pointSize: 12
-                    enabled: SystemProperties.hasWindowManager
-                    checked: !StreamingPreferences.startWindowed || !SystemProperties.hasWindowManager
-                    onCheckedChanged: {
-                        StreamingPreferences.startWindowed = !checked
+                    wrapMode: Text.Wrap
+                }
+
+                AutoResizingComboBox {
```

Comment:
It looks like this doesn't have the handling of having no window manager that the old code had

Reply:
Resolved this by setting a fallback in main.qml for when a window manager isn't present, and disabling this ability to change display mode here. Let me know if this doesn't catch the same corner case